Vizzuality/green-energy-data-platform
# Fix filter consistence when changing tabs

This PR fixes the selected filter inconsistence when changing tabs

## Testing instructions

Switch between tabs, select and deselect filters. 

- They should be consistent. 
- If a tab doesn't support filters, it should display only the Total data and no filters should be selected.
- If you go to another tab that supports filters, the last used filter should be selected.

## Jira

[GECHINA](https://vizzuality.atlassian.net/browse/GECHINA-339)
